lib/features/navigator/app_router.dart

```diff
@@ -95,6 +96,10 @@ class AppRouter extends _$AppRouter {
               path: "aboutUs",
               page: AboutUsRoute.page,
             ),
+            AutoRoute(
+              path: "guide-detail/:id",
```

Comment:
i would change to just "guide/:id" - what you think?

Reply:
That's because Stack Overflow and those 2 magic lines: 
        
` shrinkWrap: true,`
` physics: const ClampingScrollPhysics(),`

Reply:
I agree - follows convention!